compose_rl/ppo/reward_manager.py

```diff
@@ -104,13 +104,18 @@ def __init__(
             log.info(f'Initializing reward with name {reward_name}')
 
             # TODO: Validate reward_config
-            reward_cls = RL_REWARD_REGISTRY[reward_config.get('reward_type')]
+            reward_type = reward_config.pop('reward_type')
+            reward_cls = rewards_registry.get(reward_type)
 
             if issubclass(reward_cls, Reward):
                 # TODO: This assumes that all functional rewards are document level rewards.
                 # This is not necessarily true, but is a reasonable assumption for now.
                 self.granularities[reward_name] = 'document'
-                model = reward_cls(reward_config, self.tokenizer)
+                model = build_rewards(
+                    name=reward_type,
+                    tokenizer=self.tokenizer,
+                    kwargs={'cfg': reward_config},
```

Comment:
it would be nicer to flatten the 'cfg', so that the reward classes don't take in one dictionary arg called `cfg`, but instead take in named args. so it would be `kwargs=reward_config` here.

Reply:
and same in other places